uwblueprint/community-fridge-kw
# feat: create scheduling model + migration

## Notion ticket link

[Create Scheduling Table](https://www.notion.so/uwblueprintexecs/Create-Scheduling-Table-8f44e3e703cf45ecafcc0b1d27b63298)



## Implementation description
* created new model file and corresponding migration



## Steps to test
1. run migrations
2. verify that the scheduling table has been created



## What should reviewers focus on?
* 


## Checklist
